inc/planningrecall.class.php

```diff
@@ -376,7 +376,23 @@ static function cronPlanningRecall($task = null) {
       $pr = new self();
       foreach ($DB->request($query) as $data) {
          if ($pr->getFromDB($data['id']) && $pr->getItem()) {
-            if (NotificationEvent::raiseEvent('planningrecall', $pr)) {
+
+            $options = [];
+
+            //retrieve entities id from parent linked item
+            //planningrecall -> TicketTask ->  Ticket which have entity notion
+            //               -> ChangeTask ->  Change which have entity notion
+            //               -> ProblemTask -> Problem which have entity notion
+            $itemToNotify = $pr->getItem();
+            $classname = str_replace("Task", '', get_class($itemToNotify));
```

Comment:
`PlanningRecall` may be related to something else than an ITIL task, like a `Reminder`.
Code should be adapted to something like:
```php
if ($itemToNotify instanceof \CommonITILTask) {
   ...
} else {
   $linkedItem = $itemToNotify;
}
```

Reply:
Also, with https://github.com/glpi-project/glpi/pull/6224; we may also have `ProjectTask` in which one "Task" should not be removed :)